### ISSUE TITLE
Window list not alerting for steam new messages permanently

### ISSUE THREAD

**Issue by fengapappit** [linuxmint/Cinnamon#5045]
When receiving a message through Steam chat, the window in the window list flashes 3 or 4 times and then gets back to its normal state, giving the impression that no notification was received.

Linux mint : 17.3 x64
Cinnamon : 2.8.6

**Comment by Vahan86** [comment]
@fengapappit, is this still an issue for you in Cinnamon 3?

**Comment by fengapappit** [comment]
Yes, I've upgraded both Linux Mint and Cinnamon to the latest version and it's still not flashing.

**Comment by Vahan86** [comment]
@fengapappit, is this still an issue in Cinnamon 4.2?

**Comment by Vahan86** [comment]
@icarter09, can we close this issue since there is no response from OP?

**Comment by fengapappit** [comment]
My bad, didn't see your question.

I didn't notice any change in behavior, it still initially alert, then
stops.
